### DISCUSSION TITLE
So, just wondering, is it possible to use UNERF to decrypt Dragon Age Origins DLC .erf files?

### DISCUSSION THREAD

**Discussion by Traveler-VII** [xoreos/xoreos#571] [Q&A] [answered]
I've been trying to decrypt the files from the Warden's Keep DLC for the last two days, but I just can't seem to crack them. If you guys know how to, I'd love it if you could tell me, because trying to do this without the passwords is a royal pain, and the tools that could allegedly crack the DLCs are no longer available.

(Sidenote: I really despise the fact that BioWare never followed up on their promise to release the DLC source files, like they did with the base game source files. Really shows that BioWare was full of dicks, even before they stopped making good games.)

**Comment by DrMcCoy** [comment, marked as answer]
Depends on what you want to ask.

Is unerf able to decrypt the Dragon Age: Origins DLC archives provided you have the password? Yes. Unless Warden's Keep is fundamentally different from the other Origins DLC, that includes Warden's Keep.

Is unerf able to decrypt the Dragon Age: Origins DLC archives without the password? No. And any discussions in that directions aren't welcome here, for legal reasons. We do not support piracy and cracking.

I still don't know how xoreos would support the DLC in the future. How does the authorization for the DLC work, etc. But that's a different topic.
